addons/editor/functions/fnc_handleObjectEdited.sqf

```diff
@@ -0,0 +1,27 @@
+#include "script_component.hpp"
+/*
+ * Author: Ampersand
+ * Handles editing of an object by Zeus.
+ *
+ * Arguments:
+ * 0: Curator (not used) <OBJECT>
+ * 1: Edited Object <OBJECT>
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [_curator, _object] call zen_editor_fnc_handleObjectEdited
+ *
+ * Public: No
+ */
+
+params ["", "_object"];
+
+// Update building marker
+if (_object isKindOf "Building") then {
```

Comment:
Don't need this check. If the object doesn't have a marker then it won't get updated.

Reply:
https://github.com/zen-mod/ZEN/pull/704/commits/df7388c87d42a6f409e1256b36478aa186d5e9f1#diff-1a0acf5a487bf5428782a6e3f09be3f440547e468684e97b9b9890ec5bfc40dbR23